extractors/python_functions/regex_extraction/code_snippet.md

```diff
@@ -1,10 +1,11 @@
 ```python
 import re
 
+YOUR_REGEX: str = r"\$[0-9]+" # Choose any regex here
+YOUR_ATTRIBUTE: str = "text" # Choose any available attribute here
```

Comment:
only text attributes are allowed

Reply:
Did you mean for the comment beside the "text"?